stable/opa/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
This should match the version of OPA that we'll be deploying by default.
```suggestion
appVersion: 0.10.1
```

Reply:
Done.